tests/gatewayapitests/ingestion/httproute_test.go

```diff
@@ -174,3 +174,115 @@ func TestHTTPRouteWithBackendRefFilters(t *testing.T) {
 	akogatewayapitests.TeardownGatewayClass(t, gatewayClassName)
 	waitAndverify(t, gwClassKey)
 }
+func TestHTTPRouteGatewayWithEmptyHostnameInGateway(t *testing.T) {
+	gatewayClassName := "gateway-class-04"
+	gatewayName := "gateway-04"
```

Comment:
Please keep the gateway/gatewayclass/httprouteNames unique within one test file

Reply:
addressed